hack/remove_rs.sh

```diff
@@ -40,6 +42,17 @@ for QN in "${LIST[@]}"; do
       exit 1
     fi
 
+    #patch service account for push secrets
+    if [[ $NAME == *-push ]]; then
+      if kubectl patch serviceaccount $SA_NAME -n $NS -p '{"secrets": [{"name": "'$NAME'"}]}' --type="merge"
```

Comment:
I think we need it in pull-secrets section as well

Reply:
If I understood correct, must be like that:
```
secrets:
  - name: devfile-sample-go-basic-push
  - name: appstudio-pipeline-dockercfg-n6rvr
imagePullSecrets:
  - name: devfile-sample-go-basic-pull
  - name: appstudio-pipeline-dockercfg-n6rvr

```